setup.py

```diff
@@ -44,14 +44,17 @@
         'requests',
         'tensorflow-federated',
     ],
-    # TensorFlow doesn't yet support Python 3.9.
-    python_requires='>=3.6, <3.9',
+   
+    python_requires='>=3.6'
```

Comment:
I believe you need a comma at the end of this line.

Reply:
Done!